### ISSUE TITLE
Only showing first log, and no others

### ISSUE THREAD

**Issue by Nick-Lucas** [tensorflow/tensorboard#2108]
For bug reports, please include the following:

```
> conda list

tensorboard               1.13.1           py36hf484d3e_0  
tensorflow                1.13.1          gpu_py36h9b25d83_0  
tensorflow-base           1.13.1          gpu_py36h611c6d2_0  
tensorflow-estimator      1.13.0                     py_0  
tensorflow-gpu            1.13.1               h0d30ee6_0  
```

 - OS Platform and version: `Ubuntu 18.04`
 - Python version: `Python 3.6.8 :: Anaconda, Inc.`
 - For browser-related issues:
   - Browser type and version: `Chrome 73.0.3683.103 (Official Build) (64-bit)`
 
At some point my tensorboard has got stuck only able to show the first log file from any given set.

What I've tried:

* Disabling individual logs or focusing individual logs in the runs panel (as below)
* Moving a few logs to a new directory and watching that. Still the only log to be shown is whatever's first. Removing the first one and reload just gets it stuck on the next one
* Upgrade from 1.12->1.13 in conda
* Switching tensorboard to verbose logging, in case there are hidden areas: nothing shown
* Checking my JS debugger, and disabling ad-blocker: nope
* Moved out all the logs from today (when the issue appeared) in case one is corrupt: nope

I'm expecting there some cache that's become corrupt, but I couldn't find any references to this during some googling, and equally I couldn't find anything on this issue (or wasn't phrasing it correctly)

Any ideas?

![image](https://user-images.githubusercontent.com/8896153/55989053-4e9c5580-5c9d-11e9-8a62-3323738cb974.png)
![image](https://user-images.githubusercontent.com/8896153/55989074-5bb94480-5c9d-11e9-97ed-d901e33499b9.png)

**Comment by Nick-Lucas** [comment]
Quick update. I left it running for the last 15 minutes, and now there are 2 charts. The files are around 2.5-3.5gb, but it was all working really fast yesterday, so I'm a bit confused why this is taking so long to process.

**Comment by nfelt** [comment]
Hi @Nick-Lucas - sorry this fell through the cracks.  Is this issue by any chance still relevant for you?  It sounds perhaps like it could have been something transient, but if it's still happening, it'd be easiest to debug if there's a way you can share the files from the problematic directory and ideally anything you might be seeing in the verbose TensorBoard logs.

**Comment by Nick-Lucas** [comment]
Sorry I didn't update it, this should really be closed! The problem was I was recording huge amounts of data for each run, as I had switched on most of the chart types initially. It ran really fast when tensorboard had already loaded and cached the training/loss charts, but on a reboot the 3gb recording files had to be fully read in before displaying. This was definitely worsened by running off a USB HDD.

I switched off the extra recording data as I really only needed the loss charts, and everything got much faster after that.
